docs/modules/ROOT/pages/installation/binaries.adoc

```diff
@@ -12,4 +12,24 @@ We provide OS packages for RHEL/CentOS 7 & 8 as well as Debian 10 (buster).
 * EL8: https://repo.stackable.tech/repository/rpm-release/el8/
 * Debian 10: https://repo.stackable.tech/repository/deb-release
 
-Add these repositories to your OS and then install the `stackable-agent` package.
+Add these repositories to your OS, such as as suggested below, and then install the `stackable-agent` package.
+
+=== Configuring the Stackable OS package repository
+
+You will need to configure the Stackable OS package repository on the worker nodes. We’ll also take the opportunity to install OpenJDK Java 11 as well as this will be required by the Stackable services we will be running.
+
+==== Debian and Ubuntu
+    apt-get install gnupg openjdk-11-jdk curl
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 16dd12f5c7a6d76a
+    echo "deb https://repo.stackable.tech/repository/deb-release buster main" > /etc/apt/sources.list.d/stackable.list
+
+==== Red Hat and CentOS
+    /usr/bin/yum -y install gnupg2 java-11-openjdk curl
```

Comment:
I'd rather not implicitly install a jdk here, this is a fairly "large" action that is "hidden" in the section where repositories are configured.

I think we should rather add a section "preparing nodes" in which any prerequisite packages can be installed.